docs/icd.md

```diff
@@ -8,59 +8,86 @@
 
 ## Overview
 
-This document details the common software interfaces required by all services in the Arrow ecosystem.
+This document details the endpoints provided by the `svc-pricing` service.
+
+Autogenerated client-side functions for this gRPC server are conveniently packaged in the `svc-pricing-client` crate, to be used by svc-cargo, svc-rideshare, and other customer-facing services.
 
-Each service may add additional interfaces.
 
 Attribute | Description
 --- | ---
-Status | Draft
+Status | :yellow_circle: Development
 
 ## Related Documents
 
 Document | Description
 --- | ---
-:construction: Requirements & User Stories :construction: | Requirements and user stories
-
+[Pricing Model](https://docs.google.com/spreadsheets/d/1mjPtaIn3E5m7r4nyKt_sJKG9BSFm2ty7Gzo7OqERxwo) | Unit economics and pricing mechanism of flights. The core logic of `svc-pricing` is largely derived from the pricing model.
+[Uber Elevate White Paper](https://evtol.news/__media/PDFs/UberElevateWhitePaperOct2016.pdf) | Uber's research on UAM operations. Certain economic assumptions are referenced by Arrow's pricing model.
+[Services ICD](https://github.com/Arrow-air/se-services/blob/develop/docs/icd.md)
```

Comment:
Point to the svc-pricing conops and potentially the sdd

"Concept of Operations - `svc-pricing`" is how I'd label it

Reply:
[7365da3](https://github.com/Arrow-air/svc-pricing/pull/2/commits/7365da3b4afe1903cd7f32c3dc28f7d1e1a1b0ac)

The link doesn't work for now until the #3 conops pr get merged.